pkg/controller/nmstate/nmstate_controller.go

```diff
@@ -74,6 +74,12 @@ func (r *ReconcileNMstate) Reconcile(request reconcile.Request) (reconcile.Resul
 	reqLogger := log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling NMstate")
 
+	// We won't create more than one kubernetes-nmstate handler
```

Comment:
Don't we need to check also the namespace name ? I mean we should support a singleton NMstate instance named: default_namespace.defaultNMstate_name

Reply:
I think this is a cluster-wide CRD so no namespace like NodeNetworkState and friends.